templates/js-crawlee-cheerio/src/main.js

```diff
@@ -7,19 +7,27 @@
 // For more information, see https://docs.apify.com/sdk/js
 import { Actor } from 'apify';
 // For more information, see https://crawlee.dev
-import { CheerioCrawler } from 'crawlee';
-import { router } from './routes.js';
+import { CheerioCrawler, Dataset } from 'crawlee';
 
 // Initialize the Apify SDK
 await Actor.init();
 
-const startUrls = ['https://apify.com'];
+const { startUrls =  ["https://crawlee.com"], maxRequestsPerCrawl = 100 } = await Actor.getInput();
 
 const proxyConfiguration = await Actor.createProxyConfiguration();
 
 const crawler = new CheerioCrawler({
     proxyConfiguration,
-    requestHandler: router,
+    maxRequestsPerCrawl,
+    requestHandler: async ({ enqueueLinks, request, $, log }) => {
+        log.info('enqueueing new URLs');
+        await enqueueLinks( { globs: ['https://crawlee.com/*'], label: 'detail' });
```

Comment:
The `globs` can't be defined like this because it wouldn't work if you change the input to a different domain. Also the `label: 'detail'` doesn't make sense anymore as the crawler doesn't use routes